docs/Rest-API.md

```diff
@@ -0,0 +1,147 @@
+# Axe REST API
+
+Documentation of the REST API that comes along with the Axe service.
+
+## AxeResult
+
+This endpoint serves as a wrapper to collect all of the information from an axe run. 
+
+### Motivation
+
+- Provide access to results and context easily.
+- Avoid timing issues that come with multiple requests.
```

Comment:
This is the only and only reference to timing in the entire document, and the comment seems incomplete as a result without something to at least provide a one-sentence summary of how the timing issues are handled. For example, what happens if I have a multi-threaded app that makes concurrent calls to this API? If one fails, what form will that failure take?